proxy/proxy.go

```diff
@@ -108,12 +109,12 @@ func (p *Proxy) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	p.log.With("duration", duration).Info("[proxy-request] proxying complete")
 }
 
-func (p *Proxy) getMeasurementsFromTLS(conn *tls.ConnectionState) (atlsVariant variant.Variant, measurements map[uint32][]byte, err error) {
+func GetMeasurementsFromTLS(certs []*x509.Certificate, validatorOIDs []asn1.ObjectIdentifier) (atlsVariant variant.Variant, measurements map[uint32][]byte, err error) {
```

Comment:
what other purposes is it needed for to have it changed without the proxy?
Currently, I see it still only being used in the copyMeasurementsToHeader which also relies on the proxy

Reply:
It's used here: https://github.com/flashbots/cvm-reverse-proxy/pull/25